hack/update-generated.sh

```diff
@@ -38,6 +38,16 @@ go build -o "${BINDIR}/conversion-gen" k8s.io/code-generator/cmd/conversion-gen
 go mod vendor
 export GO111MODULE="off"
 
+# fake being in a gopath
```

Comment:
this is a little ugly _but_ it makes it possible to checkout kind to any directory and run this script.